src/app/authentication/auth.service.ts

```diff
@@ -0,0 +1,50 @@
+import { Injectable } from '@angular/core';
+declare var Msal: any;
```

Comment:
Why are we declaring `msal` as `any` here can't we import it ?

Reply:
We are using msal core which does not export modules. Hence the need to declare msal there...